### DISCUSSION TITLE
Icon concept for fun

### DISCUSSION THREAD

**Discussion by airamstudio** [Sequel-Ace/Sequel-Ace#1409] [Ideas] [answered]
Hi everyone in this amazing community. My name is Airam, and I'm a small Graphic Designer from Spain starting in web development. 
Not long ago I came across this wonderful alternative to the previous app SequelPro and I'm loving it so far!! I am so thankful it works in Monterrey.

The current icon is ok, but I wanted to share with you guys this icon concept I designed yesterday in my free time. Took some Inso from other native apple icons like terminal or activity monitor. Not saying it's better than the current one or that you have to use it, just wanted to share it lol.
In the picture below I included 2 color versions and some social media mockups as well as the look of the icon on the dock. I like the red one better cause that's the current icon's background color.

Hope this is ok as it doesn't really have anything to do with this apps development.

Hope you like it!!
Thank you for reading me :)

Airam Studio

<img width="1782" alt="iconbyairamstudiocopyright" src="https://user-images.githubusercontent.com/30434357/156886484-ceb02817-d4c6-4f79-bd43-a00f9718e400.PNG">

**Comment by Kaspik** [comment]
Looks great! I'm not sure we are planning to change it _right now_, but I think it would be great (if you want!) to put those into `Artwork` folder in either both the right format + source or at least the right format so anyone can download it and use it :)

Feel free to open a PR or upload them somewhere and we can include it.

We might consider changing it in the future - thanks!

**Reply by airamstudio** [reply]
Feel free to upload the icons into that folder for personal use of other people. Put them in an answer cause I don't really know how to upload them myself, sorry.

**Comment by airamstudio** [comment, marked as answer]
Here are the icons in the red version in all of the necessary sizes and .PNG format.
**Anyone can use them personally free of cost.** If you use them I would appreciate it if you give me a follow on instagram at @ airam.studio.me

If the project team decides to use this icons officially in the future, please contact me first. I will grant use free of cost.
**Commercial use of this artwork is not allowed. Modification is not allowed either.**

1024
<img width="512" alt="1024" src="https://user-images.githubusercontent.com/30434357/156891802-2dfa2f11-9d4e-49b8-8fdf-1c52d8dd2c9f.png">
512
<img width="256" alt="512" src="https://user-images.githubusercontent.com/30434357/156891803-5622e5a5-b913-49e0-a2b0-4dab67220901.png">
256
<img width="128" alt="256" src="https://user-images.githubusercontent.com/30434357/156891805-3058a4f2-156c-4ccc-b95e-df7efcc9e6a8.png">
128
<img width="64" alt="128" src="https://user-images.githubusercontent.com/30434357/156891807-5e593311-4eb5-487d-8c1f-ef8f35659a53.png">
64
<img width="32" alt="64" src="https://user-images.githubusercontent.com/30434357/156891808-40452395-3dd5-416d-b783-aaa8b7289dbf.png">
32
<img width="16" alt="32" src="https://user-images.githubusercontent.com/30434357/156891810-c4c8724a-0207-4c18-8311-eefd9e5b92e8.png">

Thank you!

**Airam Studio**
Graphic Designer & Web Dev
https://airam.studio/

**Reply by Kaspik** [reply]
Thanks, uploaded - https://github.com/Sequel-Ace/Sequel-Ace/commit/ea17aaa3e42682188a0d2520eb186f62fd4f744e

**Comment by airamstudio** [comment]
lol almost 2 years after this and this still looks sick! specially for my skills at the time haha
